uk.co.compucorp.civicrm.hrcomments/CRM/HRComments/BAO/Comment.php

```diff
@@ -19,14 +19,16 @@ public static function create($params, $validate = true) {
     if($validate){
       self::validateParams($params);
     }
-    unset($params['created_at']);
+
     unset($params['is_deleted']);
 
     $instance = new self();
     $instance->copyValues($params);
 
+    $instance->created_at = CRM_Utils_Date::processDate('now');
```

Comment:
This will always set the `created_at` date to now on updates, which I don't think is correct. I think we should throw and error if someone tries to change the date on an update